gammapy/irf/tests/test_psf_map.py

```diff
@@ -382,3 +382,51 @@ def test_to_image():
     assert_allclose(psf2D.psf_map.geom.data_shape, (1, 50, 25, 25))
     assert_allclose(psf2D.exposure_map.geom.data_shape, (1, 1, 25, 25))
     assert_allclose(psf2D.psf_map.data[0][0][12][12], 23255.41204827, rtol=1e-2)
+
+def test_psfmap_from_gauss():
+    rad = np.linspace(0, 1.5, 50)*u.deg
+    energy = np.logspace(-1, 2, 10)*u.TeV
+    energy_axis = MapAxis.from_nodes(energy, name="energy_true", interp="log", unit="TeV")
+    rad_axis = MapAxis.from_nodes(rad, name="theta", unit="deg")
+    
+    # define sigmas starting at 0.1 in steps of 0.1 deg
+    sigma = (np.arange(energy.shape[0])*0.1 +0.1)*u.deg
+    
+    # with energy-dependent sigma
+    psfmap = PSFMap.from_gauss(energy, rad, sigma)
+    assert psfmap.psf_map.geom.axes[0] == rad_axis
+    assert psfmap.psf_map.geom.axes[1] == energy_axis
+    assert psfmap.psf_map.unit == Unit("sr-1")
+    assert psfmap.psf_map.data.shape == (energy.shape[0], rad.shape[0], 1, 2)
+    assert_allclose(
+        psfmap.get_energy_dependent_table_psf().containment_radius(1*u.TeV)[0], 
+        psfmap.containment_radius_map(1*u.TeV).data[0][0]*u.deg
+    )
+    assert_almost_equal(
+        psfmap.containment_radius_map(energy[3],0.68).data[0][0]/sigma[3].value, 
+         1.5095921854516636, decimal=2
```

Comment:
If you only care about a certain precision for the reference value, you could just give the relevant decimal places...